subprojects/docs/src/samples/userguideOutput/softwareModelExtend-iv-model.out

```diff
@@ -46,7 +46,7 @@ Root project
           | Rules:
              ⤷ copyToTaskContainer
     + clean
-          | Type:   	org.gradle.api.tasks.Delete
+          | Type:   	org.gradle.language.base.internal.plugins.Clean
```

Comment:
Hm, is this strictly necessary? I mean, we are now showing an internal task as opposed to a public one.